ios/Video/RCTVideo.swift

```diff
@@ -256,6 +256,7 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        ReactNativeVideoManager.shared.registerView(newInstance: self)
```

Comment:
Why is there 2 init functions 🤔 
BTW looks good to me

Reply:
That one with coder is for compatibility with older versions of swift and storyboard - this init in general should be never called

Reply:
Removed this call. I added it just for init/deinit symmetry.